src/main/kotlin/dependencyInjection/presentationModule.kt

```diff
@@ -0,0 +1,26 @@
+package org.example.dependencyInjection
+
+import org.example.presentation.HolderCLi
+import org.koin.dsl.module
+
+val presentationModule = module {
+    single {
+        HolderCLi(
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get(),
+            get()
+        )
+    }
```

Comment:
use  singleOf(::HolderCLi)
```suggestion
 singleOf(::HolderCLi)
```

Reply:
ok